polkadot/xcm/src/lib.rs

```diff
@@ -89,11 +92,12 @@ macro_rules! versioned_type {
 		)]
 		#[codec(encode_bound())]
 		#[codec(decode_bound())]
-		#[scale_info(replace_segment("staging_xcm", "xcm"))]
```

Comment:
@franciscoaguirre Is this removal intentional? Because another pattern [few lines bellow](https://github.com/paritytech/polkadot-sdk/pull/1230/files#diff-8eb727f0e7d0dd42a8d45ab05f9450777ed6af6d22021b81d0c449e8a32aba78R186) still has `#[scale_info(replace_segment("staging_xcm", "xcm"))]`?

I think this pattern is used only for `VersionedAssetId`, so I would better add `VersionedAssetId`  to the existing test:
```
#[test]
fn ensure_type_info_is_correct() {
	let type_info = VersionedXcm::<()>::type_info();
	assert_eq!(type_info.path.segments, vec!["xcm", "VersionedXcm"]);

	let type_info = VersionedAssetId::<()>::type_info();
	assert_eq!(type_info.path.segments, vec!["xcm", "VersionedAssetId"]);
}
```